src/js_v2/domain/Car.js

```diff
@@ -0,0 +1,32 @@
+import { ERROR_MESSAGE } from '../constant/index.js';
+import { isLessThanNthCharacters } from '../util/isLessThanNthCharacters.js';
+
+export default class Car {
+  #name;
+  #position = 0;
+
+  constructor(name) {
+    this.#name = name;
+  }
```

Comment:
```suggestion
  constructor(name) {
    this.#name = new CarName(name);
  }
```
이렇게 CarName을 추출한 다음에 CarName 자체에서 검증하도록 관리해주면 어떨까요?
그럴 경우 앞서 언급한 것 처럼 CAR_NAME에 대한 상수도 `CarName` 이라는 객체 내부에서 관리해주면 되지 않을까 싶어요!